### PR TITLE
output artifacts locally with --artifacts

### PR DESCRIPTION
**Pull request by termie**

### CODE REVIEW

**Comment by termie** [comment]
i'm not totally sure on the store s3 behavior yet so need to do a little more stuffness

**Comment by termie** [comment]
yup s3 store behavior worked as expected

**Comment by faiq** [comment]
:+1:

File: docker/artifact.go [hunk 1]
```diff
@@ -38,13 +38,18 @@ type Artificer struct {
 func NewArtificer(options *core.PipelineOptions, dockerOptions *DockerOptions) *Artificer {
 	logger := util.RootLogger().WithField("Logger", "Artificer")
 
-	s3store := core.NewS3Store(options.AWSOptions)
+	var store core.Store
+	if options.ShouldStoreS3 {
+		store = core.NewS3Store(options.AWSOptions)
+	} else {
+		store = core.NewLocalStore(options.BuildPath("artifacts"))
```
Comment (faiq) [inline docker/artifact.go]: where is this method defined?

**Comment by termie** [comment]
updated per @faiq's suggestions

**Pull request merged**